crates/holochain_conductor_api/src/app_interface.rs

```diff
@@ -218,13 +218,15 @@ pub enum CellInfo {
 impl CellInfo {
     pub fn new_provisioned(
         cell_id: CellId,
+        original_dna_hash: DnaHash,
         dna_modifiers: DnaModifiers,
         name: String,
         enabled: bool,
     ) -> Self {
         Self::Provisioned(Cell {
             cell_id,
             clone_id: None,
+            original_dna_hash,
```

Comment:
@jost-s It's a bit awkward for Provisioned cells to include this, since the original DNA hash is redundant. Cell is already a bit awkward since `enabled` and `clone_id` already are only applicable to clone cells. It really feels like `Cell` should be broken out into `CloneCell` and `ProvisionedCell`.

I wouldn't block this PR on that account since at least we do have the necessary info, but in terms of a good API I think this would be a good step, especially since `StemCell` already has its own struct, and it's more similar to a provisioned call than a clone cell.